cypress/integration/mobile-learn/test-challenges.js

```diff
@@ -0,0 +1,28 @@
+import currData from '../../../config/curriculum.json';
+import { orderedSuperBlockInfo } from '../../../tools/scripts/build/build-external-curricula-data';
```

Comment:
I took a look at this import, currently only our two RWD superblocks are public in there - so it looks like these tests are visiting every page for those two. Are you going to make the rest public when you're ready? Is it excessive to visit every page? The visiting of those pages took 15 minutes on this run, if we're adding all the superblocks, it might grow to over several hours? - not sure. Perhaps we can revisit this if it becomes a problem.

Reply:
Yes, we'll open the remaining certificates one by one in the future.
I don't think it's excessive at the moment but in the future it would better to look at a parallel solution or something more efficient. The thing is we're keeping the test runner process as close as possible to how it will happen in the mobile app and the reason we aren't doing integration tests in Flutter is because it's flaky and take a long time.

Reply:
Sounds good